### DISCUSSION TITLE
`DAG.default_args.catchup` is not working...

### DISCUSSION THREAD

**Discussion by seunggabi** [apache/airflow#28015] [Q&A]
Can I make this feature?

what is different?
`default_args` vs `!default_args` 

I think, default_args support all of DAG fields

**Comment by ephraimbuddy** [comment]
`default_args` are args you want to pass to the tasks in your dag. It's not a DAG level args.

**Reply by seunggabi** [reply]
Oh I see.
but I think it is confusing about [`Operator level`](https://airflow.apache.org/docs/apache-airflow/1.10.13/_api/airflow/operators/index.html) vs `DAG level`.

`default_args` of DAG mean,
all of `DAG & Operator` args?

it is useful thing..
so I'll make pr. (this mean is just trying)

thx to your comment.

**Reply by Taragolis** [reply]
https://airflow.apache.org/docs/apache-airflow/stable/tutorial/fundamentals.html#tasks

**Reply by seunggabi** [reply]
https://github.com/apache/airflow/pull/28020/files#diff-62c8e300ee91e0d59f81e0ea5d30834f04db71ae74f2e155a10b51056b00b59bL427-L430

I think `params` same.

**Reply by Taragolis** [reply]
`default_args` from DAG propagate to DAG Tasks arguments which not explicitly set
